ecosystem/node-checker/src/server/common.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use clap::Parser;
+use url::Url;
+
+#[derive(Clone, Debug, Parser)]
+pub struct ServerArgs {
+    /// What address to listen on.
+    #[clap(long, default_value = "http://0.0.0.0")]
+    pub listen_address: Url,
+
+    /// What port to listen on.
+    #[clap(long, default_value = "20121")]
```

Comment:
btw, I think this can be default_value_t and you can use the number directly.  Is there a reason to not put the port in the URL?